tests/test_kernel_dialog.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from pytestqt.qt_compat import qt_api
+
+from pytripgui.kernel_vc import KernelController
+from pytripgui.kernel_vc import KernelQtView
+from pytrip.tripexecuter import KernelModel
+from pytrip.tripexecuter import Projectile
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+kernels = []
```

Comment:
It would be better to make a class in which the kernels will be defined. See https://docs.pytest.org/en/latest/getting-started.html#group-multiple-tests-in-a-class